.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+## Description
+[What is wrong? What the expected and current behaviours are?]
+
+
+## Code to reproduce
+[Add indicative code example to reproduce the reported case]
+
+
+## Preliminary solution
+[What should be added/changed/removed to match the expected behaviour?]
+
+
+## References
+[Add links to the related articles, packages, reference examples]
+
+
+## Environment
```

Comment:
We would also need numpy/scipy versions. Can we make a function like `skimage.print_versions()` that prints all relevant versions.